diff --git a/node_modules/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.kt b/node_modules/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.kt
index e4114c4..7ba84c1 100644
--- a/node_modules/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.kt
+++ b/node_modules/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.kt
@@ -136,7 +136,7 @@ class RNGestureHandlerButtonViewManager :
 
   @ReactProp(name = ViewProps.POINTER_EVENTS)
   override fun setPointerEvents(view: ButtonViewGroup, pointerEvents: String?) {
-    view.pointerEvents = when (pointerEvents) {
+    view.pointerEventsValue = when (pointerEvents) {
       "none" -> PointerEvents.NONE
       "box-none" -> PointerEvents.BOX_NONE
       "box-only" -> PointerEvents.BOX_ONLY
@@ -214,7 +214,8 @@ class RNGestureHandlerButtonViewManager :
 
     var exclusive = true
 
-    override var pointerEvents: PointerEvents = PointerEvents.AUTO
+    var pointerEventsValue: PointerEvents = PointerEvents.AUTO
+    override fun getPointerEvents(): PointerEvents = pointerEventsValue
 
     private var buttonBackgroundColor = Color.TRANSPARENT
     private var needBackgroundUpdate = false
